navigation/screens/forgot-password.tsx

```diff
@@ -0,0 +1,125 @@
+import React, { useState } from 'react';
+import { StyleSheet, Image, SafeAreaView, TextInput, Button, Alert, Text, View, Modal } from 'react-native';
+import { useNavigation } from '@react-navigation/native';
+
+
+function TabSubmitScreen() {
+  // State management for text inputs
+  const [email, setEmail] = useState("");
+  const [modalVisible, setModalVisible] = useState(false);
+
+  const onSubmitPress = () => {
+    // Placeholder for navigation logic
+    setModalVisible(true);
+  };
+
+  const onClosePress = () => {
+    // Placeholder for navigation logic
+    setModalVisible(false);
+  }
+
+  return (
+    <View style={styles.container}>
+      <SafeAreaView style={styles.safeArea}>
+      <Modal
+        animationType="fade"
+        transparent={true}
+        visible={modalVisible}
+        onRequestClose={() => {
+          Alert.alert('Modal has been closed.');
+          setModalVisible(!modalVisible);
+        }}>
+        
+          <View style={styles.modalView}>
+            <Text style={styles.text}>Email has been sent to {email}</Text>
+            <Button title="Close" color={"#000000"} onPress={onClosePress} />
+          </View>
+        
+      </Modal>
+        <Image
+          source={require("../../assets/images/logo-placeholder.png")}
+          style={styles.logo}
+        />
+        <Text style={styles.title}>Forgot-Password</Text>
+
+        {/* Text input fields */}
+        <TextInput
+          style={styles.input}
+          onChangeText={setEmail}
+          value={email}
+          placeholder="Email:"
+        />
+
+        {/* Submit button */}
+        <View style={styles.fixToText}>
+          <Button title="Submit" color={"#000000"} onPress={onSubmitPress} />
+        </View>
+      </SafeAreaView>
+    </View>
+  );
+}
+
+export default TabSubmitScreen;
```

Comment:
Could also just do export default in front of the function def

Reply:
changed that